### ISSUE TITLE
model.train() mode not specified in pre_train.py

### ISSUE THREAD

**Issue by siddharthKatageri** [Na-Z/attMPTI#16]
Hi,
In the training loop of `pre_train.py`, the model is not set to train mode (model.train()).
Does this affect the training process or the mode is automatically set to .train() after the evaluation is done by torch?

**Comment by Na-Z** [comment]
Thanks for pointing out this problem. It should be re-set to the *train* mode. I have added it in the latest version.
